### ISSUE TITLE
Error adding script taken from the modtweaker tutorial page...

### ISSUE THREAD

**Issue by delet3r** [jaredlll08/ModTweaker2#328]
I added a line from the modtweaker section on this site:

http://minetweaker3.powerofbytes.com/wiki/ModTweaker:Forestry_Support

And added this line to my script:

mods.forestry.Centrifuge.addRecipe([<minecraft:leaves> % 200, <minecraft:stick> % 80, <Forestry:beeCombs> % 20], 20, <minecraft:sapling>);

This is the whole script:

`

> #Name: machinerecipes.zs
> 
> #craftable iridium ore
> mods.buildcraft.AssemblyTable.addRecipe(<IC2:itemOreIridium>, 80000000, [<IC2:itemRTGPellet>, <IC2:itemMOX>]);
> mods.buildcraft.AssemblyTable.addRecipe(<Ztones:diamondZane>, 1000000, [<minecraft:diamond_pickaxe>, <minecraft:diamond_axe>, <minecraft:diamond_shovel>]);
> mods.buildcraft.AssemblyTable.addRecipe(<Ztones:terrainEater>, 10000000, [<ore:ingotEnderium>, <Ztones:diamondZane>, <ThermalFoundation:material:1024>, <ThermalFoundation:material:1024>]);
> mods.buildcraft.AssemblyTable.addRecipe(<Ztones:terrainEater>, 10000000, [<ore:ingotEnderium>, <Ztones:diamondZane>, <ThermalFoundation:material:1028>, <ThermalFoundation:material:1028>]);
> mods.buildcraft.AssemblyTable.addRecipe(<Ztones:terrainEater>, 10000000, [<ore:ingotEnderium>, <Ztones:diamondZane>, <ore:rodBlitz>, <ore:rodBlitz>]);
> 
> #Centrifuge Bauxite to get Aluminium
> mods.forestry.Centrifuge.addRecipe([<minecraft:leaves> % 200, <minecraft:stick> % 80, <Forestry:beeCombs> % 20], 20, <minecraft:sapling>);
> 
> print("Initialized 'machinerecipes.zs'");

`

When I do, it comes up with this error:

"ERROR: \scripts\machinerecipes.zs:11 > 2 methods available but none matches the parameters (minetweaker.item.WeightedItemStack[], int, minetweaker.item.IItemStack)

**Comment by Yulife** [comment]
@Voidi ?

**Comment by jaredlll08** [comment]
Well it is a documentation error... So @yulife

**Comment by Yulife** [comment]
Yeah but I'm not what @Voidi changed :)

**Comment by Voidi** [comment]
The correct function signature is `void addRecipe(WeightedItemStack[] output, IItemStack ingredient, int timePerItem)`
I don't know why the wiki is incorrect

**Comment by Yulife** [comment]
So I need to fix the wiki?

**Comment by delet3r** [comment]
Ok I think I got it.  It seems on the wiki the explanation is right, but the example code is wrong.  "//Array of Output + %, InputStack, Time in Ticks"  is what it says to explain the code, which has the cycle time last, but the example which I copied has Time, then the Input item last.  

I just switched it around, reloaded minetweaker, and it works fine.  THanks!

Also thanks for modtweaker/minetweaker and the work you guys put into this mod, I dont think id still be playing modded mc if it werent for the ability to customize like this.
